### PR TITLE
Challenge1

### PR DESCRIPTION
**Pull request by evamaina**

### CODE REVIEW

**Comment by jmnyarega** [comment]
Kindly research on how to ident HTML code, and make appropriate changes in your code.

**Comment by jmnyarega** [comment]
Make your commit message to be in present tense, e.g `Add images folder`,  `Add login page` etc

File: Designs/UI /signup.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+                    <input type="password" class="form-control" id="pwd" placeholder="Re-Enter password" name="pwd">
+                  </div>
+                  <div class="checkbox">
+                    <label><input type="checkbox" name="remember"> Remember me</label>
+                  </div>
+                  <button type="button" class="btn btn-default">Sign Up</button>
+                   <a href="User-login.html">Already member? Click here to login</a>
+                </form>
+              </div>
+</body>
```
Comment (jmnyarega) [inline Designs/UI /signup.html:34]: Add **an** extra line at the end of your file

**Comment by evamaina** [comment]
noted,thanx.working on it

File: Designs/UI /signup.html (hunk 1 above)
Comment (evamaina) [reply Designs/UI /signup.html:34]: noted thanx

**Pull request merged**